tests/virt/cluster/longevity_tests/test_multi_vm_upgrade_and_reboot.py

```diff
@@ -3,10 +3,7 @@
 from tests.virt.cluster.longevity_tests.constants import WSL2_DV_PARAMS, WSL2_VM_PARAMS
 from tests.virt.cluster.longevity_tests.utils import run_windows_upgrade_storm
 
-pytestmark = [
-    pytest.mark.usefixtures("skip_if_workers_vms", "skip_test_if_no_ocs_sc"),
-    pytest.mark.longevity,
-]
+pytestmark = [pytest.mark.usefixtures("skip_test_if_no_ocs_sc"), pytest.mark.longevity, pytest.mark.special_infra]
```

Comment:
skip_test_if_no_ocs_sc -- should fail instead of skip?

Reply:
not in scope of this pr. skips related to storage and common_cpu will be handled in separate pr

Reply:
without storage team removing programmatic skip, we can't get rid of these skips.